draft-specifications/Model.md

```diff
@@ -292,6 +360,18 @@ Promises do not support pass-invariant identity comparison.
 >
 > Tracking: https://github.com/ocapn/ocapn/issues/49
 
+Targets have [Pass Invariant Equality](#pass-invariant-equality).
+A target might be sent from a local peer to a remote peer, then the remote peer
+may send that target back to the local peer.
+The sent target will be equal to the received target.
+
+> The operator for establishing equality is specific to the implementation
+> language.
+> For JavaScript, the Pass-Invariant Equal operator is `Object.is`.
+
+Although Targets and Promises are both References for delivering messages,
+OCapn does not maintain pass invariance for promises.
```

Comment:
There is no pass-invariant promise *identity* or *equality*. However you want to put it.

As with errors, promises must round trip only to promises. This is the testable part.

The less testable part, but still part of the pass invariance of the semantics:

A promise's eventual settlement must be pass invariant. IOW, if pa in peer a, when passed to peer b results in pb, then (possibly aside from disconnection/partition trauma), pa must eventually settle iff pb eventually settles. And their settlements must be the "same" up to pass invariance.

If we allow differential disconnection/partition trauma to result in promise rejection, then we'd instead make a similar statement about fulfillment:

A promise's fulfillment settlement must be pass invariant. IOW, if pa in peer a, when passed to peer b results in pb, then, if either pa or pb eventually fulfills, the other must either eventualy fulfill to the "same" value up to pass invariance, or it must reject (with a reason indicating disconnection/partition? Do we want ocapn to stdize such a distinction?)